packages/ai/package.json

```diff
@@ -21,24 +21,25 @@
     "directory": "packages/ai"
   },
   "scripts": {
-    "lint": "yarn lint-scripts",
+    "lint": "yarn lint-misc && yarn lint-scripts",
+    "lint-misc": "prettylint --ignore-path .gitignore '**/*.{json,md,toml}'",
     "lint-scripts": "poetry run flake8",
     "test": "yarn test-units",
     "test-units": "poetry run pytest tests"
   },
   "engines": {
-    "node": ">=10.15.1",
+    "node": ">=10.15.3",
     "yarn": ">=1.13.0"
   },
   "private": true,
   "lint-staged": {
     "linters": {
-      "*.py": [
-        "poetry run flake8",
+      "*.{json,md,toml}": [
+        "prettylint --fix",
         "git add"
       ],
-      "*.toml": [
-        "prettier --write",
+      "*.py": [
+        "poetry run flake8",
```

Comment:
Minor tangent, is `flake8` the best linter for python? Do you have a reason you chose it?

Reply:
If you have a better suggestion, I'm all 👂s. I don't really have a dog in the ring as far as linters go for Python projects. I landed on `flake8` after checking against several other well supported Python projects. The majority of them were using `flake8`. It seems to work well and the plugin ecosystem seems relatively healthy, but I really don't like how it doesn't provide an auto-fix option. If you want an auto-fix option, you have to use the auto versions of the underlying libraries that `flake8` wraps. 🤷‍♂️

Reply:
No, I don't have any objections. I was jw. Thanks, dude.